superset/models/helpers.py

```diff
@@ -2112,21 +2120,22 @@ def get_sqla_query(  # pylint: disable=too-many-arguments,too-many-locals,too-ma
 
         filter_columns = [flt.get("col") for flt in filter] if filter else []
         rejected_filter_columns = [
-            col
-            for col in filter_columns
-            if col
-            and not is_adhoc_column(col)
-            and col not in self.column_names
-            and col not in applied_template_filters
-        ] + rejected_adhoc_filters_columns
+                                      col
```

Comment:
looks like our linters are behaving different, follow the steps here and you should get our standard dev tools like ruff and pylint through the pip install, and pre-commit hooks setup

https://superset.apache.org/docs/contributing/development/#installing-development-tools

Reply:
I updated using ruff and formated correctly